### ISSUE TITLE
Error output titlecase instead of lowercase

### ISSUE THREAD

**Issue by nathanborror** [grpc-ecosystem/grpc-gateway#243]
The current error output is titlecase (`{"Error": "...", "Code": "..."}`) when it should be lowercase (`{"error": "...", "code": "..."}`). This appears to be due to `errorBody` implementing `proto.Message` interface but lacking protobuf tags needed to marshal correctly. When I patch [errorBody](https://github.com/grpc-ecosystem/grpc-gateway/blob/master/runtime/errors.go#L65) with the following I end up with correct behavior:

``` go
type errorBody struct {
    Error string `protobuf:"bytes,1,name=error" json:"error"`
    Code  int    `protobuf:"bytes,2,name=code" json:"code"`
}
```

**Comment by nathanborror** [comment]
https://github.com/grpc-ecosystem/grpc-gateway/pull/244

**Comment by tmc** [comment]
This was closed by #244
